docs/DeveloperGuide.adoc

```diff
@@ -767,47 +927,92 @@ Use case ends.
 +
 Use case resumes at step 2.
 
-{More to be added}
-
 [appendix]
 == Non Functional Requirements
 
 .  Should work on any link:#mainstream-os[mainstream OS] as long as it has Java `1.8.0_60` or higher installed.
 .  Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
 .  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
-
-{More to be added}
+.  The GUI should complement the command line by providing a clean and easy to read interface.
+.  The addressbook should never encounter a single error while in use.
```

Comment:
Not sure if you can classify this as NFR. However, it is better to be here than no where!